component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/core/util/Util.java

```diff
@@ -0,0 +1,113 @@
+import org.wso2.carbon.identity.configuration.mgt.core.model.ResourceAdd;
+import org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.core.model.JWTClientAuthenticatorConfig;
+import org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.internal.JWTServiceDataHolder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.core.constant.Constants.ENABLE_TOKEN_REUSE;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.core.constant.Constants.JWT_CONFIGURATION_RESOURCE_NAME;
+
+/**
+ * Util class.
+ */
+public class Util {
+
+    /**
+     * Read the default JWT Authenticator configuration properties from the Data Holder..
+     *
+     * @return Server default {@code JWTClientAuthenticatorConfig} object.
+     */
+    public static JWTClientAuthenticatorConfig getServerConfiguration() {
+
+        JWTClientAuthenticatorConfig JWTClientAuthenticatorConfig
+                = new JWTClientAuthenticatorConfig();
+
+        JWTClientAuthenticatorConfig.
+                setEnableTokenReuse(JWTServiceDataHolder.getInstance().isPreventTokenReuse());
+        return JWTClientAuthenticatorConfig;
+    }
+
+    /**
+     * Parse Resource to JWTClientAuthenticatorConfig instance.
+     *
+     * @param resource Resource
+     * @return JWTClientAuthenticatorConfig Configuration instance.
+     */
+    public static JWTClientAuthenticatorConfig parseResource(Resource resource) {
+
+        JWTClientAuthenticatorConfig JWTClientAuthenticatorConfig
+                = new JWTClientAuthenticatorConfig();
+
+        if (resource.isHasAttribute()) {
+            List<Attribute> attributes = resource.getAttributes();
+            Map<String, String> attributeMap = getAttributeMap(attributes);
+            JWTClientAuthenticatorConfig.setEnableTokenReuse(
+                    Boolean.parseBoolean(attributeMap.get(ENABLE_TOKEN_REUSE)));
+        }
+        return JWTClientAuthenticatorConfig;
+
+    }
+
+    private static Map<String, String> getAttributeMap(List<Attribute> attributes) {
+
+        if (attributes != null) {
+            return attributes.stream().collect(Collectors.toMap(Attribute::getKey, Attribute::getValue));
+        }
+
+        return Collections.emptyMap();
+    }
+
+    /**
+     * Parse JWTClientAuthenticatorConfig to Resource instance.
+     *
+     * @param JWTClientAuthenticatorConfig Configuration Instance.
+     * @return ResourceAdd Resource instance.
+     */
+    public static ResourceAdd parseConfig(JWTClientAuthenticatorConfig JWTClientAuthenticatorConfig) {
+
+        ResourceAdd resourceAdd = new ResourceAdd();
+        resourceAdd.setName(JWT_CONFIGURATION_RESOURCE_NAME);
+        List<Attribute> attributes = new ArrayList<>();
+        addAttribute(attributes, ENABLE_TOKEN_REUSE,
```

Comment:
We don't need to pass ENABLE_TOKEN_REUSE and String.valueOf(JWTClientAuthenticatorConfig.isEnableTokenReuse() from this method. Ypo can define those inside addAttribute method itself.

Reply:
Fixed with https://github.com/wso2-extensions/identity-oauth-addons/pull/100/commits/d07baabaa4558de269ce04e390a0674e33191160